app/controllers/api/groups_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::GroupsController < Api::BaseController
+
+  def create
+    group = Group.new group_params
+    if group.save
+      render json: group, serializer: GroupBaseSerializer
+    end
```

Comment:
Could you add an `else` clause? All it needs to do is call `unprocessable_response`.